### PR TITLE
fix(deisctl): add dock to the list of commands

### PR DESCRIPTION
**Pull request by Joshua-Anderson**

### CODE REVIEW

File: deisctl/deisctl.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ Commands, use "deisctl help <command>" to learn more:
   config            set platform or component values
   refresh-units     refresh unit files from GitHub
   ssh               open an interacive shell on a machine in the cluster
```
Comment (mboersma) [inline deisctl/deisctl.go]: Could you fix the existing typo here while you're at it? "interactive"

**Comment by Joshua-Anderson** [comment]
@mboersma Done!

**Comment by carmstrong** [comment]
Code LGTM.

**Comment by mboersma** [comment]
Code LGTM.

**Pull request merged**